database/postgres/postgres.go

```diff
@@ -25,6 +26,7 @@ func init() {
 }
 
 var DefaultMigrationsTable = "schema_migrations"
+var DefaultStatementTimeout = 10000 // 10 seconds
```

Comment:
Maybe let's not add such a short default statement time out? Since this makes a breaking change it should rather be much higher? Wdyt

Reply:
Also 
```suggestion
var DefaultStatementTimeoutMs = 10000 // 10 seconds
```

Reply:
No migration should run for 10s but if we feel so we can definitely bump this up.

Reply:
One case where migrations can run for longer is if concurrent indices are used which theoretically can run for long durations. But if we add an explicit flag we can subvert this issue.